### PR TITLE
{tools}[system/system] rclone v1.56.0

### PR DESCRIPTION
**Pull request by scicomp-moffitt**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/rclone/rclone-1.56.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - https://hprc.tamu.edu
+
+easyblock = 'MakeCp'
+
+name = 'rclone'
+version = '1.56.0'
+#versionsuffix = '-amd64'
```
Comment (lexming) [inline easybuild/easyconfigs/r/rclone/rclone-1.56.0.eb:7]: Please keep the version suffix for this easyconfig as the binaries are specific to `amd64`.

File: easybuild/easyconfigs/r/rclone/rclone-1.56.0.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+description = """
+ Rclone is a command line program to sync files and directories to and from
+ a variety of online storage services
+"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://downloads.%(name)s.org/v%(version)s/']
+sources = ['%(name)s-v%(version)s-linux-amd64.zip']
+#checksums = ['c52cbf3646a2d15765b87cf05fc3b2bca3b1d2782d4922046c597bd979e42720']
```
Comment (lexming) [inline easybuild/easyconfigs/r/rclone/rclone-1.56.0.eb:20]: Checksums are missing.

**Review by lexming — CHANGES_REQUESTED** [review]

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/009a94d7ea0092d2343579dab7ec5cd6 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/ec37dc22fc247cbfff9f27dd232f00e1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3103.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/09d24b4212490e99dd8cfa96a4aa9e63 for a full test report.

**Review by lexming — APPROVED** [review]
@scicomp-moffitt thanks for the update!

LGTM

**Comment by lexming** [comment]
@boegelbot please test @ generoso

**Comment by lexming** [comment]
Going in, thanks @scicomp-moffitt !

**Comment by lexming** [comment]
@boegelbot please test @ generoso

**Pull request merged**